independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/api/WebClientApplicationException.java

```diff
@@ -17,19 +29,150 @@ public WebClientApplicationException() {
     }
 
     public WebClientApplicationException(String reason) {
-        super(reason);
-    }
-
-    public WebClientApplicationException(Throwable t) {
-        super(t);
-    }
-
-    public WebClientApplicationException(String reason, Throwable t) {
```

Comment:
Is it on purpose that you removed those two constructors?

Reply:
Yup, they were not being used anywhere.

Now the argument could be made that they might be used in used code, so if you want, I can add them back

Reply:
I dunno, I don't remember who calls that. :shrug:

Reply:
My take is that we should remove them as no one is calling them at the time being. Also RR is still marked as experimental so it would be best to remove them now than drag them around forever

Reply:
OK.